test/integration/trialBalance.js

```diff
@@ -0,0 +1,96 @@
+/* global expect, chai, agent */
+/* jshint expr : true */
+
+const helpers = require('./helpers');
+const uuid = require('node-uuid');
+
+/*
+ * The /trial_balance API endpoint
+ */
+describe('(/trial) API endpoint', function () {
```

Comment:
It would be helpful if this text used the full endpoint name `/trail_balance` instead of just `(/trail)`.  I can see that being confusing later.